puck.config.tsx

```diff
@@ -324,11 +327,27 @@ export const config = {
       },
       render: TextEditor,
     },
+    Excalidraw: {
+      fields: {
+        text: {
+          type: "custom",
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          render: ExcalidrawEditor as any,
+        },
+      },
+      defaultProps: {
+        text: {
+          raw: "",
+          html: "<div>Excalidraw's default content</div>",
+        },
+      },
+      render: Excalidraw,
+    },
   },
   categories: {
     text: {
       title: "Text",
-      components: ["TextEditor", "Link", "List", "Typography"],
+      components: ["TextEditor", "Excalidraw", "Link", "List", "Typography"],
```

Comment:
**suggestion**: I think that excalidraw is different category, probably interactive or media.

Reply:
Okay, will leave it uncategorized for now.